src/services/asset-inventory/AssetInventoryContainer.vue

```diff
@@ -40,8 +40,15 @@ export default defineComponent({
         VerticalPageLayout,
     },
     setup() {
-        registerServiceStore<AssetInventoryState>('assetInventory', assetInventoryStoreModule, assetInventoryStore);
+        const cloudServicePageStore = useCloudServicePageStore();
+        const cloudServiceDetailPageStore = useCloudServiceDetailPageStore();
+
         const { breadcrumbs } = useBreadcrumbs();
+
+        onUnmounted(() => {
+            cloudServicePageStore.$dispose();
+            cloudServiceDetailPageStore.$dispose();
+        });
```

Comment:
Question: Don't you need to the reset data?

Reply:
It doesn't seem necessary, but that would be safer!